cfe_internal/core/watchdog/templates/watchdog-windows.ps1.mustache

```diff
@@ -0,0 +1,23 @@
+$LOGFILE="{{{logfile}}}"
+
+$long_running_cf_agent_threshold_min = 5
+$long_running_cf_agent_count = @(Get-Process cf-agent -erroraction "silentlycontinue" | Where { $_.StartTime -lt (Get-Date).AddMinutes(-$long_running_cf_agent_threshold_min) }).count
+$long_running_cf_agent_count_threshold = 1
+$running_cf_agent_count = @(Get-Process cf-agent -erroraction "silentlycontinue").count
```

Comment:
nitpick: you don't use this variable anywhere before assigning to it again on line 20, so you can just remove this from here :)

Reply:
ack